aqua/wezterm/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           cargo   1.0
 PortGroup           github  1.0
 
-github.setup        wez wezterm 20240203-110809-5046fc22
+github.setup        wez wezterm 30345b36d8a00fed347e4df5dadd83915a7693fb
```

Comment:
The only issue here is that the SHA version doesn't follow the same format as the version there currently.
So maybe we can do this, but implicitly set a datestamped current version with the assumption that the next release will have a higher (later) datestamp.

So perhaps something like this:
```
github.setup        wez wezterm 30345b36d8a00fed347e4df5dadd83915a7693fb
version             20240813
```

`20240813` is the datestamp of that commit.

Reply:
Not a bad plan. We should decide if this is the SHA we want, too. I want to stress that this was just HEAD at the time I tested to see if it had been fixed upstream, and I've just run with it so far.